<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comp_cpu</class>
 <widget class="QDialog" name="comp_cpu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU选择</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res/res.qrc">
    <normaloff>:/img/curve.png</normaloff>:/img/curve.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>组件信息</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_compname"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>组件名称：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>组件ID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_compID"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>441</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>CPU选择</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>105</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>请选择发送至：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>192.168.1.12</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>请选择CPU：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_cpuchoose">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>0012</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0013</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0014</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>请选择核心：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_corechoose">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>00xx-18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>00xx-19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>00xx-20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>00xx-20</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Res/res.qrc"/>
 </resources>
 <connections/>
</ui>
